zebra-state/src/service/finalized_state/disk_format.rs

```diff
@@ -249,6 +251,20 @@ impl IntoDisk for orchard::tree::Root {
     }
 }
 
+impl IntoDisk for ValueBalance<NonNegative> {
+    type Bytes = [u8; 32];
+
+    fn as_bytes(&self) -> Self::Bytes {
+        self.to_bytes()
+    }
+}
+
+impl FromDisk for ValueBalance<NonNegative> {
+    fn from_bytes(bytes: impl AsRef<[u8]>) -> Self {
+        ValueBalance::from_bytes(bytes.as_ref().try_into().unwrap())
```

Comment:
This seems to be a common practice in this file. 

https://github.com/ZcashFoundation/zebra/blob/d9964f764ad173fb5716d070b9d8875bbaed5aa3/zebra-state/src/service/finalized_state/disk_format.rs#L146
https://github.com/ZcashFoundation/zebra/blob/d9964f764ad173fb5716d070b9d8875bbaed5aa3/zebra-state/src/service/finalized_state/disk_format.rs#L194
...

I think they are fine but should we consider an alternative ?

Reply:
Let's just do the same thing as the rest of the file for now.